auth_mellon_handler.c

```diff
@@ -191,6 +191,9 @@ static char *am_generate_metadata(apr_pool_t *p, request_rec *r)
    <SingleLogoutService\n\
      Binding=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect\"\n\
      Location=\"%slogout\" />\n\
+   <SingleLogoutService\n\
+     Binding=\"urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST\"\n\
+     Location=\"%slogout\" />\n\
```

Comment:
This is added with `%s` but an extra variable is not added in r215. `/mellon/metadata` now gives internal server error.